lib/ffmpeg/qsv/transcoder.py

```diff
@@ -20,10 +20,14 @@ class TranscoderTest(BaseTranscoderTest):
       "avc" : dict(
         sw = (dict(maxres = (16384, 16384)), have_ffmpeg_decoder("h264"), "h264"),
         hw = (platform.get_caps("decode", "avc"), have_ffmpeg_decoder("h264_qsv"), "h264_qsv"),
+        va_hw = (platform.get_caps("decode", "avc"), have_ffmpeg_decoder("h264"), "h264"),
```

Comment:
I see we already accepted "va_hw" in gst xcode.  So, I guess this is ok.

Later, I may refactor all xcode design since xcode tests are getting a bit messy.